certified-connectors/Power Textor/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6782 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Power Textor",
+    "description": "With the Power Textor connector, send SMS messages to your customers and get push notifications when a text message is received. Build brand loyalty by a one-on-one conversation with customers with Power Textor actions that support a range of text message sending options; including on-demand SMS, scheduled SMS, event reminders, and Google Review requests. Each functionality is available for an individual number or a group of numbers.",
+    "version": "1.0",
+    "contact": {
+      "name": "Power Textor Support",
+      "url": "https://powertextor.com/#contact",
+      "email": "info@powertextor.com"
+    }
+  },
+  "host": "api-powertextor-v2.azurewebsites.net",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/contacts/count": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "Error": {
+                  "type": "string",
+                  "description": "Error"
+                },
+                "Data": {
+                  "type": "object",
+                  "properties": {
+                    "CampaignId": {
+                      "type": "string",
+                      "description": "CampaignId"
```

Comment:
@MohsinAQ All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)